web/app/js/components/MetricsTable.jsx

```diff
@@ -19,12 +19,12 @@ import { processedMetricsPropType, successRateWithMiniChart } from './util/Metri
 */
 const smMetricColWidth = "70px";
 
-const withTooltip = (d, metricName) => {
+const withTooltip = (d, metricName, hideUnit) => {
   return (
     <Tooltip
       title={metricToFormatter["UNTRUNCATED"](d)}
       overlayStyle={{ fontSize: "12px" }}>
-      <span>{metricToFormatter[metricName](d)}</span>
+      <span>{ !hideUnit ? metricToFormatter[metricName](d) : metricToFormatter["NO_UNIT"](d)}</span>
```

Comment:
I think for better readability, it might be good to invert this statement.

```js
hideUnit ? metricToFormatter["NO_UNIT"](d) : metricToFormatter[metricName](d) 
```

Reply:
removed!